zenith/src/main.rs

```diff
@@ -18,13 +18,13 @@ use walkeeper::defaults::{
 };
 use zenith_utils::auth::{Claims, Scope};
 use zenith_utils::postgres_backend::AuthType;
-use zenith_utils::zid::{ZTenantId, ZTimelineId};
+use zenith_utils::zid::{ZNodeId, ZTenantId, ZTimelineId};
 use zenith_utils::GIT_VERSION;
 
 use pageserver::branches::BranchInfo;
 
 // Default name of a safekeeper node, if not specified on the command line.
-const DEFAULT_SAFEKEEPER_NAME: &str = "single";
+const DEFAULT_SAFEKEEPER_ID: ZNodeId = 1;
```

Comment:
IMO it is pretty scary to have a default for it. If we miss I’d in cli  by accident we will have different nodes with the same id